src/sqs/sqs.ts

```diff
@@ -284,7 +284,9 @@ export class SqsQueue<MessagePayload> {
 	}
 
 	private async handleSqsMessageExecutionError(err, context: SQSMessageContext<MessagePayload>) {
+		const unsafeLogger = getLogger("message-error-handler-unsafe", { level: "warn", unsafe: true });
```

Comment:
Tagging a logger with `unsafe: true` this will push logs to [env]-unsafe where we are allowed to write out UGC and PII.... within reason

Reply:
Within reason? Do we have any idea what that is? Might be worth throwing a HOWTO runbook together to get the team on the same page and so we have somewhere to point future Arc folks to.

Reply:
Fair poinst, there is a few docs floating around, most of them we collected in this page https://hello.atlassian.net/wiki/spaces/OTFS/pages/1649328545/Logging+UGC+PII

but I might reference it or make a mini runbook, so its a bit more digestable.

Basically there is still restricted data like credit cards and such that are big no noes

Reply:
Makes sense 😄